src/common/hooks/use-do-change-screen.ts

```diff
@@ -0,0 +1,29 @@
+import { useDispatch } from 'react-redux';
+import { doChangeScreen as changeScreen } from '@store/onboarding/actions';
+import { ScreenPaths } from '@store/onboarding/types';
+import { useNavigate } from 'react-router-dom';
+import { useCallback } from 'react';
+
+type doChangeScreen = (path: ScreenPaths, changeRoute?: boolean) => void;
+
+export function useDoChangeScreen(): doChangeScreen {
```

Comment:
```suggestion
type UseDoChangeScreen = (path: ScreenPaths, changeRoute?: boolean) => void;

export function useDoChangeScreen(): UseDoChangeScreen {
```
Typical pattern is to pascal case types